app/views/tools/index.html.erb

```diff
@@ -1 +1,13 @@
-<h1>Welcome to the index page</h1>
+<h1>Tools in your area</h1>
+
+<div class="container">
+  <div class="cards">
+    <% @tools.each do |tool| %>
+    <div class="tool-card">
+      <%= tool.name %> <br>
+      <%= tool.description %> <br>
+      £<%= tool.price %> per hour
+    </div>
+    <% end %>
```

Comment:
I have this created already

Reply:
sure, I could not see it on the main, have you created yours in a pull already?

Reply:
> sure, I could not see it on the main, have you created yours in a pull already?

I hava pr for that already